docs/metrics.md

```diff
@@ -124,39 +109,44 @@ Services.telemetry.scalarSet(
 	"category.scalar_name", value
 );
 ```
-e.g. `lockboxV1.datastoreCount` for the scalar name.
+e.g. `lockbox-addon.loginCount` for the scalar name.
 
 We can also use `scalarAdd` to increment a scalar value by some amount.
 
 ## Scalar Metrics
 
-These are the metrics we currently collect regarding the state of user datastores.
+These are the metrics we currently collect regarding the state of the user's login storage.
+
+- `loginCount` (SCALAR_TYPE_COUNT). Current count of the number of items in the user's login storage. If possible, this scalar should be set upon opening either the doorhanger or full extension interface.
```

Comment:
We can definitely trigger this ping when the user opens the doorhanger or loads the management page.

Would it also be useful to check this value if the user doesn't interact with Lockbox? For instance, we could check this value at browser startup, or periodically (whenever Firefox is open).

Reply:
Relaying from an earlier IRL conversation when I asked exactly this: these metrics are also measuring as user's engagement with Lockbox specifically, so it's preferable to limit to triggers specific to interacting with Lockbox.

Reply:
Yes +1 to Matt, trying to limit pings with extension data to those who are actually using it (makes counting things like "active" users a bit easier).